codecov.yml

```diff
@@ -0,0 +1,8 @@
+coverage:
```

Comment:
Looks good! Lets add a comment in this file to explain why we set the targets to 0%

Reply:
I Added a comment.